### ISSUE TITLE
Simplify running chain of commands in single line

### ISSUE THREAD

**Issue by roman-petrov** [invertase/melos#234]
Currently we can run chain of commands like:

`melos run <command1> && melos run <command2> && melos run <command3>`

or even

`melos <command1> && melos <command2> && melos <command3>`

Can it be even simpler like: `melos <command1> <command2> <command3>` ?

**Comment by spydon** [comment]
I don't think this is doable since the trailing commands could be interpreted as input to command1.
